### DISCUSSION TITLE
I want to know if this is possible 🤔

### DISCUSSION THREAD

**Discussion by Dhyfer1** [martinet101/ElevenClock#1190] [General]
Hi @marticliment 

I was wondering if it's possible for the clock in Elevenclock, in 24-hour format, to display times in this way:

| for AM | for PM | 
|----------|----------|
|00:00 AM in exchange of 12:00 AM|00:00 PM in exchange of 12:00 PM|
|01:00 AM in exchange of 1:00 AM|01:00 PM in exchange of 13:00 PM|
|02:00 AM in exchange of 2:00 AM|02:00 PM in exchange of 14:00 PM|
|03:00 AM in exchange of 3:00 AM|03:00 PM in exchange of 15:00 PM|
|04:00 AM in exchange of 4:00 AM|04:00 PM in exchange of 16:00 PM|
|05:00 AM in exchange of 5:00 AM|05:00 PM in exchange of 17:00 PM|
|06:00 AM in exchange of 6:00 AM|06:00 PM in exchange of 18:00 PM|
|07:00 AM in exchange of 7:00 AM|07:00 PM in exchange of 19:00 PM|
|08:00 AM in exchange of 8:00 AM|08:00 PM in exchange of 20:00 PM|
|09:00 AM in exchange of 9:00 AM|09:00 PM in exchange of 21:00 PM|
|10:00 AM doesn't change|10:00 PM in exchange of 22:00 PM|
|11:00 AM doesn't change|11:00 PM in exchange of 23:00 PM|

As you can see, I want to keep the hours always from 00 to 11 no matter if it's AM or PM. Do you think it's possible?

**Comment by marticliment** [comment]
Hello, I think you will be stuck with 12 pm for 00 am and 12 am for 00 pm.

To change that, either set your system format to 24h and restart your pc or try setting a custom time format on ElevenClock.

**Reply by Dhyfer1** [reply]
Ok. I had to change the format to 12 hours, no need to restart the PC, the change is instantaneous. And I used a custom time format in Elevenclock as you said, and I found a bug when I wanted to use the "Use a custom line height" function. When I changed values in this function, there was no visible change, but when I deactivated the function "Set a custom date and time format (for advanced users only)" it worked "Use a custom line height".

That's all, thanks for your solution
